crowdin_api/api_resources/screenshots/resource.py

```diff
@@ -56,6 +59,7 @@ def add_screenshot(
         name: str,
         projectId: Optional[int] = None,
         autoTag: Optional[bool] = None,
+        labelIds: Optional[Iterable[int]] = None
```

Comment:
please also include the other missing parameters (fileId, branchId, directoryId)